examples/reference_environments/s3/run_quickstart.py

```diff
@@ -0,0 +1,60 @@
+import great_expectations as gx
```

Comment:
I don't think so. The feedback from devrel was that the notebook version was preferred. Let's discuss, I am actually thinking of removing this. I thought maybe it would help in testing, but I think it would be better to just run the notebook with an assert rather than try to keep both in sync.

Reply:
Yeah on second thought, I will remove it in this PR. We can always grab it out of the commit or recreate if we want it later.